package.json

```diff
@@ -14,9 +14,12 @@
     "dev": "turbo run dev",
     "format": "prettier --check \"**/*.{ts,tsx,md}\"",
     "lint": "eslint --ext .tsx,.ts ./packages",
+    "percy": "yarn ./apps/storybook/ exec percy storybook ./build",
     "prepare": "husky install",
     "release-packages": "yarn build && yarn changeset publish",
+    "storybook:build": "yarn ./apps/storybook/ exec build-storybook -c .storybook -o ./build",
```

Comment:
you should just be able to use a filter with turbo to run this command

`yarn turbo run build --filter=@project44-manifest/storybook`